src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  flex-direction: row;
+  align-items: center;
+  font-family: Roboto, sans-serif;
+  padding-left: 50px;
+  padding-right: 50px;
+  margin: 0;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
```

Comment:
Don't use tag names for styling (except html and body)
<img width="821" alt="image" src="https://github.com/user-attachments/assets/b16a11eb-7d68-42f6-b868-e4acfb3529db">

Reply:
> Don't use tag names for styling (except html and body) <img alt="image" width="821" src="https://private-user-images.githubusercontent.com/63243136/374080064-b16a11eb-7d68-42f6-b868-e4acfb3529db.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg5MTIyNTgsIm5iZiI6MTcyODkxMTk1OCwicGF0aCI6Ii82MzI0MzEzNi8zNzQwODAwNjQtYjE2YTExZWItN2Q2OC00MmY2LWI4NjgtZTRhY2ZiMzUyOWRiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE0VDEzMTkxOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU1Y2UyZGIyNzMyNDY4NGRhY2MyOTRkZmQ3N2RiOWQ0YjZkYTk0NDZhNzQ2YWY3OWNjOWRmMTEzZjgyOGM4MzgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.40GvVH5jSsDBtoK1t9VTjjQDBlkvE9blyxQQQQpO5gk">

Not fixed